docs/reference/setup/install/docker.asciidoc

```diff
@@ -404,15 +404,41 @@ file is obfuscated but not encrypted. If you want to encrypt your
 bind-mount it to the container as
 `/usr/share/elasticsearch/config/elasticsearch.keystore`. In order to provide
 the Docker container with the password at startup, set the Docker environment
-value `KEYSTORE_PASSWORD` to the value of your password. For example, a `docker
-run` command might have the following options:
+value `KEYSTORE_PASSWORD` to the value of your password. For example, a `docker run` 
+command might have the following options:
 
 [source, sh]
 --------------------------------------------
--v full_path_to/elasticsearch.keystore:/usr/share/elasticsearch/config/elasticsearch.keystore
+-v full_path_to/config:/usr/share/elasticsearch/config
 -E KEYSTORE_PASSWORD=mypassword
 --------------------------------------------
 
+If the keystore is mounted incorrectly, it will induce example Docker errors from attempting to
```

Comment:
@jrodewig I'm not sure about this sentence, can you take a look?

Reply:
Thanks for the ping @pugnascotia! I'll take a look and push some changes with edits.

Thanks for putting this together @stefnestor. Some updates are definitely needed here!